docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -931,17 +954,17 @@ quarkus.oidc.logout.backchannel.path=/back-channel-logout
 The absolute `back-channel logout` URL is calculated by adding `quarkus.oidc.back-channel-logout.path` to the current endpoint URL, for example, `http://localhost:8080/back-channel-logout`.
 You will need to configure this URL in the admin console of your OIDC provider.
 
-You will also need to configure a token age property for the logout token verification to succeed if your OpenID Connect Provider does not set an expiry claim in the current logout token.
+You will also need to configure a token age property for the logout token verification to succeed if your OpenID Connect provider does not set an expiry claim in the current logout token.
 For example, set `quarkus.oidc.token.age=10S` to ensure that no more than 10 seconds elapse since the logout token's `iat` (issued at) time.
 
 [[front-channel-logout]]
 ===== Front-channel logout
 
 You can use link:https://openid.net/specs/openid-connect-frontchannel-1_0.html[Front-channel logout] to log out the current user directly from the user agent, for example, its browser.
-It is like <<back-channel-logout,Back-channel logout>> but the logout steps are executed by the user agent like the browser and not in the background by the OIDC provider.
+It is similar to <<back-channel-logout,Back-channel logout>> but the logout steps are executed by the user agent, such as the browser, and not in the background by the OIDC provider.
 This option is rarely used.
 
-You can configure Quarkus to support `Front-Channel Logout` as follows:
+You can configure Quarkus to support front-channel logout as follows:
```

Comment:
```suggestion
You can configure Quarkus to support Front-channel logout as follows:
```

Reply:
I also updated the earlier reference then from "back-channel logout" to "Back-channel logout" (line 942)